src/ice_sdk.c

```diff
@@ -20,11 +20,12 @@ static void init_rgb_led(void) {
 }
 
 void ice_sdk_init(void) {
-    init_rgb_led();
+    //init_rgb_led(); //this can be called by the user. Otherwise interferes with FGPA executiong bitstream
```

Comment:
I should've explain better. The init_rgb_led() should be called explicitly (opt-in) by the sdk consumer.
Otherwise if the FPGA is using the same shared leds the behavior then sdk should provide an easy way to unset those leds.